### ISSUE TITLE
Errors on LiveView mount are reported twice

### ISSUE THREAD

**Issue by crbelaus** [elixir-error-tracker/error-tracker#104]
When raising an error on the LiveView mount hook it gets reported twice and generates two occurrences instead of one. The occurrences are generated from the following Phoenix telemetry events:
- `[:phoenix, :router_dispatch, :exception]`
- `[:phoenix, :live_view, :mount, :exception]`

**Comment by tjarratt** [comment]
Is this due to the fact that mount is called twice, once for the initial page view and again when the websocket connects ?

**Comment by crbelaus** [comment]
This is correct @tjarratt. I meant to close this issue past week but then I forgot about it.

Since mount is called twice, the errors are reported twice as well. Just like any function throwing an error would report the error every time it is called.
